jest.config.js

```diff
@@ -28,11 +28,7 @@ module.exports = {
   collectCoverage: true,
   coverageDirectory: 'coverage',
   coverageReporters: ['json', 'text'],
-  collectCoverageFrom: [
-    'src/**/*.{ts,tsx}',
-    '!src/**/*.d.ts',
-    '!src/**/index.{js,ts}',
-  ],
+  collectCoverageFrom: ['src/**/*.{ts,tsx}', 'src/client/filtering/*.js', '!src/**/*.d.ts', '!src/**/index.{js,ts}'],
```

Comment:
There are other JS files we may want coverage on but for now I only added a single new directory